src/routes/wallet/p2sh/tx-proposal.routes.js

```diff
@@ -98,6 +99,18 @@ txProposalRouter.post(
   buildCreateTokenTxProposal,
 );
 
+txProposalRouter.post(
+  '/mint-tokens',
+  body('token').isString().notEmpty(),
+  body('amount').isInt({ min: 1 }).toInt(),
+  body('address').isString().notEmpty().optional(),
```

Comment:
Is it optional? The api docs description says it isn't.

But if the idea is to copy what we have in the current mint-tokens api, then it should be optional and the api docs must be fixed (the current mint tokens api docs as well).

Reply:
It was fixed in the api docs.

Reply:
See comment: https://github.com/HathorNetwork/hathor-wallet-headless/pull/312#discussion_r1261545722